openmls/src/ciphersuite/signable.rs

```diff
@@ -153,6 +153,7 @@ pub trait Verifiable: Sized {
                 return Err(SignatureError::VerificationError);
             }
         };
+        // https://validation.openmls.tech/#valn1301
```

Comment:
Are these in https://github.com/openmls/validation.dhall/pull/20?

Reply:
Exactly